frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/copy_and_paste/custom_cut_command.dart

```diff
@@ -24,6 +26,13 @@ CommandShortcutEventHandler _cutCommandHandler = (editorState) {
     return KeyEventResult.ignored;
   }
 
+  final context = editorState.document.root.context;
+  if (context == null || !context.mounted) {
+    return KeyEventResult.ignored;
+  }
+
+  context.read<ClipboardState>().didCut();
```

Comment:
Do all the document pages contain this provider? If you're not sure, using '?' is safer.